lib/membrane_flv_plugin/format.ex

```diff
@@ -38,32 +21,57 @@ defmodule Membrane.FLV do
           | :MP3_8k
           | :device_specific
 
-  @video_codec BiMap.new(%{
-                 2 => :sorenson_h263,
-                 3 => :screen_video,
-                 4 => :vp6,
-                 5 => :vp6_with_alpha,
-                 6 => :screen_video_2,
-                 7 => :H264
-               })
-
   @typedoc """
   List of video codecs supported by the FLV format.
   """
   @type video_codec_t() ::
           :sorenson_h263 | :screen_video | :vp6 | :vp6_with_alpha | :screen_video_2 | :H264
 
   @spec index_to_sound_format(non_neg_integer()) :: audio_codec_t()
-  def index_to_sound_format(index), do: BiMap.fetch!(@sound_format, index)
+  def index_to_sound_format(0), do: :pcm
+  def index_to_sound_format(1), do: :adpcm
+  def index_to_sound_format(2), do: :MP3
+  def index_to_sound_format(3), do: :pcmle
+  def index_to_sound_format(4), do: :nellymoser_16k_mono
+  def index_to_sound_format(5), do: :nellymoser_8k_mono
+  def index_to_sound_format(6), do: :nellymoser
+  def index_to_sound_format(7), do: :g711_a_law
+  def index_to_sound_format(8), do: :g711_mu_law
+  def index_to_sound_format(10), do: :AAC
+  def index_to_sound_format(11), do: :Speex
+  def index_to_sound_format(14), do: :MP3_8k
+  def index_to_sound_format(15), do: :device_specific
```

Comment:
Nice. However, I think I would add another function clause to provide a nicer error that 'no function clause matching'